account/manager.go

```diff
@@ -1316,10 +1399,8 @@ func (m *manager) RenewAccount(ctx context.Context,
 	// rest of the flow. This should request a signature from the auctioneer
 	// and assuming it's valid, broadcast the update transaction.
 	modifiers = append(modifiers, StateModifier(StatePendingUpdate))
-	modifiedAccount, spendTx, err := m.spendAccount(
-		ctx, account, packet, spendWitnessType, modifiers, false,
-		bestHeight,
-	)
+	modifiedAccount, spendTx, err := m.spendAccount(ctx, account, RENEW, packet,
```

Comment:
nit: don't change the formatting for a multi-line method call, here and a few other places in this file.

Reply:
Ah not intentional, IDE formatting. Will fix.